### DISCUSSION TITLE
Parallel states

### DISCUSSION THREAD

**Discussion by BrunoUY** [statelyai/xstate#2722] [Q&A] [answered]
In the docs I see that in the section of "Parallel states" "child states" are also known as "regions". Just a basic question:
in a compound state, a child state is also known as a "region", or the region name is only under a "parallel state" ?

thank you, Bruno.

**Comment by davidkpiano** [comment, marked as answer]
Good question: A direct child of a parallel state is known as a "region" in state machine literature, and this only applies to parallel states.

Calling it a "region" is also helpful (IMO) because it distinguishes between child states of a compound state, of which only one can be active, and a "region" child state of a parallel state, which is always active by default when the parent parallel state is active and represents a region of child states where only one is active.
